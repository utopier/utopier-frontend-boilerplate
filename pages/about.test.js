import React from 'react';
import { render } from '@testing-library/react';
import About from './about';

describe('<About/>', () => {
  it('matches snapshot', () => {
    const utils = render(<About />);
    expect(utils.container).toMatchSnapshot();
  });
  it('has h1 tag', () => {
    const utils = render(<About />);
    utils.getByText('About');
  });
});
