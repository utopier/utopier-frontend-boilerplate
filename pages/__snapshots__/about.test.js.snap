// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> matches snapshot 1`] = `
<div>
  <div>
    <h1>
      About
    </h1>
  </div>
</div>
`;
